### PR TITLE
enhance test cases for gaussian_random_op and uniform_random_op

### PR DESCRIPTION
**Pull request by pengli09**

### CODE REVIEW

File: python/paddle/v2/framework/tests/test_gaussian_random_op.py [hunk 1]
```diff
@@ -6,29 +6,66 @@
...
-        self.assertAlmostEqual(numpy.mean(tensor), .0, delta=0.1)
-        self.assertAlmostEqual(numpy.std(tensor), 1., delta=0.1)
+        tensor = numpy.array(scope.find_var("Out").get_tensor())
+        tensor1 = numpy.copy(tensor)
+
+        op.run(scope, context)
+        tensor2 = numpy.array(scope.find_var("Out").get_tensor())
+
+        false_num = (tensor1 != tensor2).sum()
+        self.assertGreater(false_num, int(0.95 * 1000 * 784))
```
Comment (QiJune) [inline python/paddle/v2/framework/tests/test_gaussian_random_op.py]: I am curious about why the rate is `0.95`

Comment (pengli09) [reply python/paddle/v2/framework/tests/test_gaussian_random_op.py]: This is a personal choice. Although tensor1 and tensor2 are sampled randomly, some values may still be identical by chance. So I choose a not-so-big, not-so-small value. I'll add a comment for it.

**Comment by pengli09** [comment]
Add comments for the ratio 0.95 used in `uniform_random_random_seed_test()` and `gaussian_random_random_seed_test()`.

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献代码。由于Paddle V1/V2版本已不再维护，相关代码也已从develop分支上删除，因此关闭您的PR，欢迎您向Paddle最新版-Fluid贡献代码。
Thanks for contributing to PaddlePaddle! Since V1/V2 will not be maintained anymore, and related codes have been deleted from develop branch as well, we close this PR. Welcome to contribute to Fluid——the latest version of PaddlePaddle.

**Pull request closed without merging**